patch.py

```diff
@@ -2,10 +2,17 @@
 from typing import List
 from patch_utils import *
 
-full_patch: List[Addition] = []
+with open('Version.txt', 'r') as f:
```

Comment:
I can't see any file named Version.txt, where does it come from ?

Reply:
https://github.com/google/or-tools/blob/stable/Version.txt